### PR TITLE
[#40] Update theme of default app

### PR DESCRIPTION
**Pull request by tavareshansen**
[ Contents ]
- Added theme utility manage methods
- Defined types for theme
- Implemented the way for switching theme
- Added base theme with colors defined in design

[ ToDo ]
- Need to discuss with @monikappv re style details. (e.g. font-size consistency, states of buttons, etc)

fixes: #40

### CODE REVIEW

**Comment by abhishekgahlot** [comment]
Looks good to me @tavareshansen , Shall we deploy it first somewhere so that @monikappv can verify?

**Comment by tavareshansen** [comment]
@abhishekgahlot 
There're various font-sizes in design, which I'd like to confirm with @monikappv .
cc @rufuspollock

**Comment by tavareshansen** [comment]
@rufuspollock please review this PR.

**Comment by rufuspollock** [comment]
@tavareshansen the preview does not seem to show the new design at least in terms of images etc. Is that intentional?

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,5 @@ yarn-error.log*
 .env.development.local
 .env.test.local
 .env.production.local
+
+pages/App.css
```
Comment (rufuspollock) [inline .gitignore:33]: why do we ignore this - a small comment could be useful

**Pull request merged**